docs/ops.html

```diff
@@ -1314,7 +1314,125 @@
     </tbody>
   </table>
 
-  <h5><a id="others_monitoring" href="#others_monitoring">Others</a></h5>
+
+
+  <h4><a id="kafka_streams_monitoring" href="#kafka_streams_monitoring">Kafka Streams Metrics</a></h4>
+
+  A Kafka Streams instance contains all the producer and consumer metrics as well as additional metrics specific to streams. By default Kafka Streams has metrics with two recording levels: debug and info. The debug level records all metrics, while the info level records only some of them.  Use the following configuration option to specify which metrics you want collected:
```

Comment:
Mention that embedded producer and consumers also have their own metrics, and provide the default client-id field value as well.